### PR TITLE
fix(docs): update form screenshots as per latest web browser.

### LINKED ISSUE #26951: Update Browser Screenshots to recent versions in Form Examples (Learn: Basic native form controls)
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/Basic_native_form_controls

### What specific section or headline is this issue about?

Single line text fields

### What information was incorrect, unhelpful, or incomplete?

I suggest updating the browser screenshots for all the Web Forms section pages since the browser version is about 40 versions behind.

Updating the content to discuss how things are currently working instead of how they worked before.

Please see the attached screenshot as an example of the older browser versions.

mdn - I suggest updating the browser screenshots for all the Web Forms section pages since the browser version is about 40 versions behind

### What did you expect to see?

I expect to see updated screenshots of browser version examples.

### Do you have any supporting links, references, or citations?

nope

### Do you have anything more you want to share?

nope

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/basic_native_form_controls`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/Basic_native_form_controls
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/basic_native_form_controls/index.md
* Last commit: https://github.com/mdn/content/commit/65b9418c7d0e3a331ac50249adf0024f44789923
* Document last modified: 2023-05-09T00:34:38.000Z

</details>

### PR DESCRIPTION
**Pull request by abhishekrnjn**
### Description

Update Browser Screenshots to recent versions in Form Examples. 

### Motivation

Docs will get updated to current version of browser screenshots

### Related issues and pull requests

Fixes #26951

### CODE REVIEW

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Here is a basic single line text field example:
 
 Single line text fields have only one true constraint: if you type text with line breaks, the browser removes those line breaks before sending the data to the server.
 
-_The following screenshot shows default, focused and disabled text input types in Firefox 71 and Safari on macOS and in Chrome 79 and Edge 18 on Windows 10._
+_The following screenshot shows default, focused and disabled text input types in Firefox 114 and Safari on macOS and in Chrome 114 and Edge 114 on Windows 11._
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:68]: Drive-by comment/suggestion: Stating the browser and OS versions here creates the dependency to always maintain/update the text and screenshot going forward. How about we remove the version numbers?

```suggestion
_The following screenshot gives you an idea of how the text input field looks in its various states – by default, when it has focus, and when it is marked disabled – across Firefox and Safari on macOS and in Chrome and Edge on Windows._
```

I would even argue if adding this screenshot here is really helpful (why do we need to show both Chrome and Edge) and perhaps we should remove the screenshot and the accompanying text altogether. But if it has to be retained, I think we should make the text generic as suggested above to reduce maintenance.

Comment (hamishwillee) [reply files/en-us/learn/forms/basic_native_form_controls/index.md:68]: @dipikabh Thanks very much. My take on this is that in a learn topic it is useful to explain what the controls look like and how they might differ. However given that all browsers do focus, disabled, default somewhat similarly I'm not sure that showing all of the different browsers is worth it. Further, 
- this is going to be an ongoing maintenance issue- will have to do this again in a number of releases.
- there are a bunch of other images further down in the doc.

How about we replace all the images with images from one browser, which we don't identify, and have words to describe what you're seeing. So "mocked up", something like:

----

The screenshot below shows a text input in default, focused, and disabled states. Most browsers indicate focus with a focus ring around the control and the disabled state using grey text or a faded/semi-opaque control.

image

----

And do something similar for the other controls.

Thoughts?

Comment (abhishekrnjn) [reply files/en-us/learn/forms/basic_native_form_controls/index.md:68]: So from discussion, what i understand is
- we need screenshot of default, focused and disabled state from only one browser (will prefer Chrome on macOS).
- screenshot and text changes need to be done for all the images present in doc.
These changes i can do.

Comment (hamishwillee) [reply files/en-us/learn/forms/basic_native_form_controls/index.md:68]: Yes, except:

> screenshot and text changes need to be done for all the images present in doc.

All sections where you're showing off a feature with UI should have an image - even if there is not an image in the section currently (so not necessarily "update the current pics" - there might be more.

Yes, just one browser. Might be worth saying on the first use that these are from macOS but other browsers use similar highlighting techniques.

**Review by hamishwillee — CHANGES_REQUESTED** [review]
Just adding "request for changes" as per https://github.com/mdn/content/pull/27436#discussion_r1257580872

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 2]
```diff
@@ -66,9 +66,11 @@ Here is a basic single line text field example:
 
 Single line text fields have only one true constraint: if you type text with line breaks, the browser removes those line breaks before sending the data to the server.
 
-_The following screenshot shows default, focused and disabled text input types in Firefox 71 and Safari on macOS and in Chrome 79 and Edge 18 on Windows 10._
+_The screenshot below shows a text input in default, focused, and disabled states. Most browsers indicate focus with a focus ring around the control and the disabled state using grey text or a faded/semi-opaque control._
 
-![Screenshot of the disabled attribute and default :focus styles on a text input in Firefox, Safari, Chrome and Edge.](disabled.png)
+![Screenshot of the default, focused and disabled states text input in Chrome on macOS](disabled.png)
+
+_Screenshots used in this document are taken in Chrome 115 on macOS. There could be little changes when these fields/buttons are displayed in different browser with different versions, but basic highlighting technique is similar._
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:73]:
```suggestion
The screenshots used in this document were taken in the Chrome browser on macOS. There may be minor variations in these fields/buttons across different browsers, but the basic highlighting technique remains similar._
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 3]
```diff
@@ -143,9 +149,9 @@ Related checkbox items should use the same [`name`](/en-US/docs/Web/HTML/Element
 </fieldset>
 ```
 
-The following screenshots show default, focused and disabled checkboxes in Firefox 71 and Safari 13 on macOS and Chrome 79 and Edge 18 on Windows 10:
+The following screenshot shows default, focused and disabled checkboxes. Default and disabled checkboxes are in checked state where as focused checkbox is in unchecked state with focus ring around it.
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:152]:
```suggestion
The following screenshot shows checkboxes in the default, focused, and disabled states. Checkboxes in the default and disabled states appear checked, whereas in the focused state, the checkbox is unchecked, with focus ring around it.
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 4]
```diff
@@ -181,9 +187,9 @@ Several radio buttons can be tied together. If they share the same value for the
 </fieldset>
 ```
 
-The following screenshots show unchecked and checked radio buttons, radio buttons that have focus, and disabled unchecked and checked radio buttons — on Firefox 71 and Safari 13 on macOS and Chrome 79 and Edge 18 on Windows 10.
+The following screenshot shows default and disabled radio buttons in checked state along with focused radio button in unchecked state.
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:190]:
```suggestion
The following screenshot shows default and disabled radio buttons in the checked state, along with focused a radio button in the unchecked state.
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 5]
```diff
@@ -181,9 +187,9 @@ Several radio buttons can be tied together. If they share the same value for the
...
 
-![Radio buttons on Firefox 71 and Safari 13 on Mac and Chrome 79 and Edge 18 on Windows 10](radios.png)
+![Radio buttons in chrome 115 on macOS](radios.png)
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:192]:
```suggestion
![Default, focused and disabled Radio buttons in chrome 115 on macOS](radios.png)
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 6]
```diff
@@ -270,9 +276,9 @@ Below you can find examples of each button `<input>` type, along with the equiva
 
 Buttons always behave the same whether you use a {{HTMLElement("button")}} element or an {{HTMLElement("input")}} element. As you can see from the examples, however, {{HTMLElement("button")}} elements let you use HTML in their content, which is inserted between the opening and closing `<button>` tags. {{HTMLElement("input")}} elements on the other hand are {{glossary("void element", "void elements")}}; their displayed content is inserted inside the `value` attribute, and therefore only accepts plain text as content.
 
-The following examples show default, focused, and disabled button input types — in Firefox 71 and Safari 13 on macOS and Chrome 79 and Edge 18 on Windows 10.
+The following examples show default, focused, and disabled button input types.
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:279]:
```suggestion
The following screenshot shows a button in the default, focused, and disabled states. In the focused state, there is a focus ring around the button, and in the disabled state, the button is greyed out.
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 7]
```diff
@@ -270,9 +276,9 @@ Below you can find examples of each button `<input>` type, along with the equiva
...
 
-![Default, focused and disabled button input types in Firefox 71 and Safari 13 on Mac and Chrome 79 and Edge 18 on Windows 10](buttons.png)
+![Default, focused and disabled button input types in chrome 115 on macOS](buttons.png)
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:281]:
```suggestion
![Default, focus, and disabled button states in chrome 115 on macOS](buttons.png)
```

File: files/en-us/learn/forms/basic_native_form_controls/index.md [hunk 8]
```diff
@@ -319,6 +325,10 @@ On some mobile devices, the file picker can access photos, videos, and audio cap
 <input type="file" accept="audio/*;capture=microphone" />
 ```
 
+The following screenshot shows File picker widget in default, focused and disabled state when no file is selected.
+
+![File picker widget in chrome 115 on macOS](filepickers.png)
```
Comment (dipikabh) [inline files/en-us/learn/forms/basic_native_form_controls/index.md:330]:
```suggestion
The following screenshot shows the file picker widget in the default, focus, and disabled states when no file is selected.

![File picker widget in default, focus, and disabled states in chrome 115 on macOS](filepickers.png)
```

**Review by dipikabh — DISMISSED** [review]
Hello @abhishekrnjn, thanks a lot for making these updates as suggested. I've added a few edits.

Apologies for the delay on reviewing these changes.

**Comment by abhishekrnjn** [comment]
Hello @dipikabh, I have made minor changes in suggestions, rest suggestions looks correct to me. 
Shall i commit these suggested changes?

**Review by hamishwillee — APPROVED** [review]
@abhishekrnjn So what would normally happen is that you would look at @dipikabh suggestions and merge those you think are better than what they replace, or discuss why not. I've reviewed those and made only one minor typo fix and merged.

This is good to go in. IMO it is a good update - it is a lot less complex while retaining all the essential information. Thank you very much for working on this (and @dipikabh for the reviews).

Note, I'm merging this because the review team is short staffed and I know @dipikabh has other responsibilities this week.

**Comment by dipikabh** [comment]
Thanks for your review support as well @hamishwillee on this. Thank you for committing the suggestions 👍

**Pull request merged**